app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Details of the Selected post by User</h1>
\ No newline at end of file
+<h1>User Details</h1>
+<%= render 'partials/user', user: @user %>
+
+<div class="bio-card container">
+  <h3 class="bio-card-header">Bio</h3>
+  <p><%= @user.bio %> </p>
+</div>
+
+<% @recent_posts.each_with_index do |post, index| %>
+  <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+  <div class="detailed-post-box container">
+  <div class="username-and-counters">
+    <p class="dp-username">Post #1 by <%= @user.name %></p>
+    <small class="dp-counters">Comments: <%= @post.comments.length %>, Likes: <%= @post.likes.length %></small>
```

Comment:
- [ ] when I click on a user with a post and the post has comments, I get this error, kindly fix it
![Screenshot from 2023-07-27 09-11-00](https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/66097371/79910545-3d3d-4c41-ae3e-5b8e681f9452)